packages/env/lib/cli.js

```diff
@@ -124,6 +124,11 @@ module.exports = function cli() {
 				coerce: parseXdebugMode,
 				type: 'string',
 			} );
+			args.option( 'xhprof', {
+				describe:
+					'Enables XHProf. If not passed, XHProf is turned off.',
+				type: 'string',
```

Comment:
If there's only one XHProf mode, I think this should be a boolean instead.

Reply:
XHProf does support different configuration modes, though I'm unsure how important it is to support that config through the CLI versus as an option passed to the profiler when it's loaded. See: https://github.com/joemcgill/wp-profiler/blob/trunk/plugin.php#L19-L24 for an example of how these flags can be set via code rather than configuration.

Reply:
Good point. I think if people will have to configure this code in the first place, we don't need to support setting the options via flags. If we did find a way to automate the wp-config.php changes, then it'd make sense to add them to the flags.